src/Image/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { CSSProperties } from 'react';
+
+interface ImgProps {
+    alt: string;
+    height?: number;
+    src: string;
+    title: string;
+    width?: number;
+}
+
+interface AmpImgProps extends ImgProps {
+    layout: string;
+}
+
+// TODO: Get rid of the anys
+interface ImageProps extends ImgProps {
+    globalSettings: any;
+    utils: any;
+}
+
+const responsiveImgStyles: CSSProperties = {
+    width: '100%'
+};
+
+const AmpImg = (props: AmpImgProps) => <amp-img {...props} />;
+const Img = (props: ImgProps) => (
+    <picture>
+        <img
+            {...props}
+            style={props.height ? undefined : responsiveImgStyles}
```

Comment:
Instead of making this conditional on the `height` attribute, you can use `max-width` instead of width. That will cause the image to display at its native size unless it is wider than it's parent.